proto/spire/go.mod

```diff
@@ -1,3 +1,9 @@
 module github.com/spiffe/spire/proto/spire
 
 go 1.13
+
+require (
```

Comment:
Was this inclusion accidental?

Reply:
Hmm, the protobuf and grpc deps seem good actually since the repo contains generated code that uses those packages. The spire dependency is in there because of the catalog plugin code.

We added this because we wanted to get at a node.proto from a 3rd party project without having to pull in SPIRE but this will pull in SPIRE anyway....

We really need to separate this out cleanly.

Reply:
I think this can go in as-is. We can revisit what we want to do with the module separately.

Reply:
oops

I'm fine with having this removed, totally accidental from running the `build.sh` setup-related things (though ideally running `make` or `build.sh` commands should result in no change at HEAD)